lrauv_system_tests/CMakeLists.txt

```diff
@@ -44,3 +44,11 @@ target_link_libraries(${PROJECT_NAME}_support PUBLIC
   lrauv_ignition_plugins::lrauv_state
   gtest)
 target_compile_features(${PROJECT_NAME}_support PUBLIC cxx_std_17)
+#============================================================================
+# Tests
+if(BUILD_TESTING)
+  enable_testing()
+  include(Dart)
```

Comment:
Are we calling DART directly anywhere in our tests? If not, can this be removed.

Reply:
Amended in 937013dacc73ba4c2581150af8585b480f713006.